### ISSUE TITLE
Deployment Tasks: Restart service leaves service stopped

### ISSUE THREAD

**Issue by JLJTGR** [webprofusion/certify#530]
Version 5.1.6.0

This appears to be a regression as it worked properly at one point.  I'm not sure when this started happening, but it could have been months ago.  I seem to remember one of my services being stopped for an unknown reason, and this would fit the bill.

----
```
2020-09-13 17:00:13.823 -04:00 [INF] ---- Performing Task [On-Demand or Manual Execution] :: Restart WAMP64 ----
2020-09-13 17:00:13.823 -04:00 [INF] Task [Restart WAMP64] :: Task is enabled and primary request was successful.
2020-09-13 17:00:13.826 -04:00 [INF] Service already stopped [wampapache64] 
2020-09-13 17:00:13.826 -04:00 [INF] Service Restarted
``` 
Service remains stopped.

----
```
2020-09-13 17:34:30.575 -04:00 [INF] ---- Performing Task [On-Demand or Manual Execution] :: Restart WAMP64 ----
2020-09-13 17:34:30.576 -04:00 [INF] Task [Restart WAMP64] :: Task is enabled and primary request was successful.
2020-09-13 17:34:30.577 -04:00 [INF] Stopping service [wampapache64] 
2020-09-13 17:34:34.482 -04:00 [INF] Service Restarted
``` 
Service is stopped and is not started.

----

**Comment by webprofusion-chrisc** [comment]
Thanks, nothing has changed recently on the service restarts, so perhaps we need to be more robust with retries etc. As a workaround you could add an additional task to start the service, so that it tries to start the service twice. Really we need to poll the status and try again with some exponential backoff for wait, as some services will take a long time to start back up.

**Comment by JLJTGR** [comment]
Please notice that it never says `Starting service [wampapache64] `.  I do not believe it is even trying to start the service.

This is what it looked like the last time it was working...
```
2020-08-06 18:21:45.541 -04:00 [INF] Task [Restart WAMP64] :: Task is enabled and primary request was successful.
2020-08-06 18:21:45.550 -04:00 [INF] Stopping service [wampapache64] 
2020-08-06 18:21:50.504 -04:00 [INF] Starting service [wampapache64] 
2020-08-06 18:21:51.912 -04:00 [INF] Service Restarted
2020-08-06 18:21:51.913 -04:00 [INF] Deploy to Apache KEY/CERT :: Task Completed OK
```

**Comment by webprofusion-chrisc** [comment]
Argh, found it. Update incoming shortly.

**Comment by webprofusion-chrisc** [comment]
Fixed in 5.1.7, thanks for identifying this issue! The problem was caused by a combination of general refactoring for deployment tasks parameters and failure to maintain current integration tests for the service manager task (fixed now).
